DemoApp/DemoApp/DemoApp.csproj

```diff
@@ -50,6 +50,7 @@
 
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="7.0.0" />
+		<PackageReference Include="Mopups" Version="1.1.1" />
```

Comment:
Do we still need the Mopups reference? Haven't we migrated to own implementation?

Reply:
Removed